src/core_modules/capture-core/rulesEngineActionsCreator/runRulesForSingleEvent.js

```diff
@@ -94,35 +70,11 @@ function getDataElements(program: Program) {
     return getRulesEngineDataElementsAsObject(dataElements);
 }
 
-function runRulesEngine(
-    rulesEngine: RulesEngine,
-    programRulesContainer: ProgramRulesContainer,
-    dataElementsInProgram: { [elementId: string]: DataElementForRulesEngine },
-    orgUnit: OrgUnit,
-    optionSets: ?OptionSets,
-    currentEventData: ?EventData | {} = {},
```

Comment:
It is important to keep this default value ({}). If you don't supply a value some event context variables will be missing when running the rules.